### PR TITLE
contacts_v0.1

### PR DESCRIPTION
**Pull request by valerasam**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
-    );
-  }
+  return (
+    <div className="App">
+      <Switch>
+        <Route path='/home' component={Home} />
+        <Route path='/component' component={Home} />
+        <Route path='/contact' component={Contact} />
+        <Route path='/contact-list' component={ContactList} />
+        <Redirect from='/' to='/home' />
```
Comment (AlexBychkov) [inline src/App.js]: think, it's better to move <Redirect from='/' to='/home' /> to the bottom of Switch to have it more readable

**Review by AlexBychkov — APPROVED** [review]

**Review by foxprogs — APPROVED** [review]

File: src/pages/contact/Contact.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+
+
+class Contact extends Component {
+    render() {
+        const { contactData: { name, phone, email, id } } = this.props;
+        return (
+            <TableRow>
+                <TableCell>
```
Comment (AlexBychkov) [inline src/pages/contact/Contact.js:12]: it's beter to have this in one line like in lines 15 or 16

File: src/pages/contact/Contact.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { TableCell, TableRow } from '@material-ui/core';
+
+
```
Comment (AlexBychkov) [inline src/pages/contact/Contact.js:4]: lets' remove all except one new lines

File: src/components/menuUser/MenuUser.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+
+  //style for link
```
Comment (AlexBychkov) [inline src/components/menuUser/MenuUser.js:19]: let's move styles to the bottom of this component

File: src/components/header/Header.js [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft'; 
+import MenuIcon from '@material-ui/icons/Menu'; 
+import MenuUser from '../menuUser/MenuUser';
+import { FirstListMenu, SecondListMenu } from './listMenu/ListMenu';
+
+
+const drawerWidth = 240;
+
+const useStyles = makeStyles((theme) => ({
```
Comment (AlexBychkov) [inline src/components/header/Header.js:14]: let's move styles to a separate file, like in PR number 6

**Review by AlexBychkov — CHANGES_REQUESTED** [review]

**Pull request merged**